packages/fluentui/docs/src/examples/components/Toolbar/Types/index.tsx

```diff
@@ -6,6 +6,7 @@ const Types = () => (
   <ExampleSection title="Types">
     <ComponentExample
       title="Default"
+      toolbarAriaLabel="Example default"
```

Comment:
we can probably do a little bit better with this aria label and the ones before. `Default` ... does not give much information. `default toolbar example` is probably better. I would go through all the `aria-labels` and make some improvements.

Reply:
We have already on the Toolbar component 31 toolbars and the idea recently was following:
![image](https://user-images.githubusercontent.com/22620150/80490791-2ba11a00-8962-11ea-8494-9e1acf4d492c.png)

Each toolbar contains the buttons ("Try it, Theme it, RTL, Popout") related to the example will start to with the example wording and following the "example name." To be able distinguish if you cannot see page to which toolbar is "example toolbar" related. 

It is not ideal to have in the toolbar name the wording "Toolbar" because then the narration is following:
"{toolbar aria label} toolbar " and if we already keep word 'Toolbar' in aria-label then it narrate 'Toolbar' twice. 

But unfortunately in some cases if the name of the example is "Toolbar can contain a menu" I rather let it there because the when people will do testing I hope they will find it easier the toolbar because title of toolbar example is as well the heading. 

But I am open any naming improvement here :) just only we need to consider that we have 
- toolbar contains the buttons ("Try it, Theme it, RTL, Popout") 
- toolbar which is as example itself 
And this toolbars are following in the DOM one after another...